generator.py

```diff
@@ -264,7 +264,7 @@
 
 def generate_html_elements(ctx, n):
     for i in range(n):
-        tag = random.choice(_HTML_TYPES.keys())
+        tag = random.choice(list(_HTML_TYPES.keys()))
```

Comment:
You can go even simpler and drop the `.keys()`

Reply:
thanks for remind me